com.microsoft.mrtk.graphicstools.unity/Samples~/Clipping Primitives/LightMaps/LightingData.asset.meta

```diff
@@ -0,0 +1,8 @@
+fileFormatVersion: 2
```

Comment:
Would you mind deleting all the lightmap data? If you don't need it we normally don't check it in to reduce space.